tests/foreman/cli/test_bootdisk.py

```diff
@@ -90,3 +90,46 @@ def _finalize():
     # Check if full-host bootdisk can be downloaded.
     bootdisk = module_target_sat.cli.Bootdisk.host({'host-id': host['id'], 'full': 'true'})
     assert 'Successfully downloaded host disk image' in bootdisk['message']
+
+
+def test_positive_bootdisk_subnet_download(
+    module_location, module_sca_manifest_org, module_target_sat
+):
+    """Verify satellite is able to doanload subnet disk image.
+
+    :id: ed6da37f-ba3f-43c9-9169-40c818e42136
+
+    :steps:
+        1. Create a subnet
+        2. Download the subnet disk image
+        3. Check if subnet disk image is downloaded properly
+
+    :expectedresults: subnet disk image is downloaded properly
+
+    :Verifies: SAT-23549
+
+    :customerscenario: true
+    """
+    capsule = module_target_sat.nailgun_smart_proxy
+    name = gen_string('alpha')
+    mask = '255.255.255.0'
+    network = gen_ipaddr()
+    domain = module_target_sat.cli_factory.make_domain()
+    gateway = gen_ipaddr()
+    ipam_type = SUBNET_IPAM_TYPES['dhcp']
+    subnet = module_target_sat.cli_factory.make_subnet(
+        {
+            'name': name,
+            'mask': mask,
```

Comment:
I think we're creating subnet and other stuff in the default org and location here, so can we update the test run with module org and location? or delete the created stuff in teardown?

Reply:
Updated to use module org and loc